templates/index.html

```diff
@@ -3,55 +3,165 @@
+        <div class="card bg-primary rounded-lg text-white mb-2" style="width: 18rem;">
+          <div class="card-body">
+            <p class="card-text">
+              Hi Emily! I'm sure you already know that the Georgia runoff election, the one that
+              determines who controls the Senate, is on January 5th. I just wanted to pass along
+              this link to find your early voting location -- you can start voting today!<br><br>
+              <strong>www.voteamerica.com/where-to-vote</strong></p>
+          </div>
+        </div>
+
+        <!-- TODO: See if this works, I have an Android rn -->
+        <a href="sms://body=I%27m%20sure%20you%20already%20know%20that%20the%20Georgia%20runoff%20election%2C%20the%20one%20that%20determines%20who%20controls%20the%20Senate%2C%20is%20on%20January%205.%20I%20just%20wanted%20to%20pass%20along%20this%20link%20to%20find%20your%20early%20voting%20location%20--%20you%20can%20start%20voting%20today%21%20https%3A%2F%2Fwww.voteamerica.com%2Fwhere-to-vote%2F"
+          target="_blank" rel="noopener" class="btn text-white mb-3"
+          style="background-color: #e12814;">Text a friend</a>
       </div>
-      <div class="col-md-4 col-sm-12 mb-3">
-        <label for="input-last">County</label>
-        <input type="text" class="form-control" placeholder="County" name="county" id="input-county">
+
+      <div>
+        <div class="card rounded-lg mb-2" style="width: 18rem;">
+          <div class="card-body">
+            <p class="card-text">
+              Hi Emily! I'm sure you already know that the Georgia runoff election, the one that
+              determines who controls the Senate, is on January 5th. I just wanted to pass along
+              this link to find your early voting location -- you can start voting today!<br><br>
+              <strong>www.voteamerica.com/where-to-vote</strong></p>
+          </div>
+        </div>
+
+        <a href="mailto:?to=&subject=Vote%20early%20in%20the%20Georgia%20runoff%20election&body=I%27m%20sure%20you%20already%20know%20that%20the%20Georgia%20runoff%20election%2C%20the%20one%20that%20determines%20who%20controls%20the%20Senate%2C%20is%20on%20January%205.%20I%20just%20wanted%20to%20pass%20along%20this%20link%20to%20find%20your%20early%20voting%20location%20--%20you%20can%20start%20voting%20today%21%20https%3A%2F%2Fwww.voteamerica.com%2Fwhere-to-vote%2F"
+          target="_blank" rel="noopener" class="btn text-white mb-3"
+          style="background-color: #e12814;">Email a friend</a>
       </div>
-      <div class="col-md-4 col-sm-12 mb-3">
-        <label for="input-last">City</label>
-        <input type="text" class="form-control" placeholder="City" name="city" id="input-city">
+
+      <hr class='my-5'>
+    </div>
+
+    <div class='mt-4'>
+      <h4>3️⃣ Follow up!</h4>
+      <p>Research shows that people vote when you ask them to. Make sure to follow up and ask your
+        friends if they've voted!</p>
```

Comment:
feels like it would be good to give a little more contextualization to the search form. something like this, maybe:

```suggestion
        friends if they've voted! If you like, you can also use this form to search for them in the Georgia early voting database.</p>
```